qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -310,11 +310,11 @@
         <module name="ClassTypeParameterName"/>
         <module name="ConstantName"/>
         <module name="LocalFinalVariableName">
-            <property name="format" value="^[a-z]{3,12}$"/>
+            <property name="format" value="^[a-z]{2,12}$"/>
```

Comment:
@jrdalpra this is not correct, we should allow ONLY `id` from 2 char variables, all the rest should follow restriction of `{3,12}`